geniza/annotations/tests/test_annotations_views.py

```diff
@@ -8,11 +8,61 @@
 from pytest_django.asserts import assertContains, assertNotContains
 
 from geniza.annotations.models import Annotation
-from geniza.annotations.views import AnnotationResponse
+from geniza.annotations.views import AnnotationLastModifiedMixin, AnnotationResponse
 from geniza.corpus.models import Document
 from geniza.footnotes.models import Footnote, Source, SourceType
 
 
+class TestAnnotationLastModifiedMixin:
+    def test_get_etag(self, annotation):
+        mixin = AnnotationLastModifiedMixin()
+        # nonexistent annotation should return None
+        assert mixin.get_etag(request=None, pk=1234) == None
+        # otherwise should return ETag
+        assert mixin.get_etag(request=None, pk=annotation.pk) == annotation.etag
+
+    def test_get_last_modified(self, annotation):
+        mixin = AnnotationLastModifiedMixin()
+        # nonexistent annotation should return None
+        assert mixin.get_last_modified(request=None, pk=1234) == None
+        # otherwise should return last modified
+        assert (
+            mixin.get_last_modified(request=None, pk=annotation.pk)
+            == annotation.modified
+        )
+
+    def test_disaptch(self, admin_client, annotation):
```

Comment:
typo in your test method name :-) 
```suggestion
    def test_dispatch(self, admin_client, annotation):
```